awscli/help.py

```diff
@@ -114,6 +114,11 @@ def _convert_doc_content(self, contents):
 
     def _send_output_to_pager(self, output):
         cmdline = self.get_pager_cmdline()
+        if not self._exists_on_path(cmdline[0]):
+            LOG.debug("Pager '%s' not found in PATH, printing raw help." %
+                      cmdline[0])
+            print(output.decode('utf-8'))
```

Comment:
We should use one of the compat module's print functions to handle py2/3 differences for stdout.